packages/neuron-ui/src/components/Receive/index.tsx

```diff
@@ -10,33 +11,73 @@ declare global {
   }
 }
 
+const AddressPanel = styled.div`
+  dispaly: flex;
+  display: -webkit-flex; /* Safari */
```

Comment:
unnecessary, styled-component will add this prefix

Reply:
Without `-webkit-flex`, it don't work.

Reply:
Interesting, the styled-component should add that automatically.

Leave it, it's running only in chrome, prefix is unnecessary.